app/templates/administrative-units/administrative-unit/ministers/new.hbs

```diff
@@ -82,9 +82,9 @@
                   of probeer een andere zoekopdracht. Indien een persoon
                   ontbreekt, meldt dit aan
                   <a
-                    href="mailto:TODO"
+                    href="mailto:organisaties.abb@vlaanderen.be"
```

Comment:
Since this is used in multiple pages I think it might be better to store this in the environment.js file and use the config helper to retrieve it.

Quick summary:
1. add a `contactEmail` or `helpDeskEmail` to the environment.js file, similar to this:
https://github.com/lblod/frontend-contact-hub/blob/6c2b2c90c6fbfb67328fcb6db0c903582df291eb/config/environment.js#L5
2. replace all the hardcoded emails with the config that can be retrieved with the config helper like this:
https://github.com/lblod/frontend-contact-hub/blob/6c2b2c90c6fbfb67328fcb6db0c903582df291eb/app/templates/application.hbs#L6
3. I think this also needs to be changed in the contact page:
https://github.com/lblod/frontend-contact-hub/blob/6c2b2c90c6fbfb67328fcb6db0c903582df291eb/app/controllers/contact.js#L4
This can be removed and replaced with a config import:

```js
import config from 'frontend-contact-hub/config/environment';

// ...

contactEmail = config.contactEmail;
```